services/static-webserver/client/source/class/osparc/data/model/Node.js

```diff
@@ -1327,31 +1327,36 @@ qx.Class.define("osparc.data.model.Node", {
       }
     },
 
-    __waitForServiceReady: function(srvUrl) {
-      // ping for some time until it is really ready
-      fetch(srvUrl)
-        .then(request => {
-          /*
-          if (request.status >= 200 || request.status < 300) {
-            this.__waitForServiceWebsite(srvUrl)
-          }
-          */
+    __waitForServiceReady: async function(srvUrl) {
+      const retry = () => {
+        this.getStatus().setInteractive("connecting");
+        // Check if node is still there
+        if (this.getWorkbench().getNode(this.getNodeId()) === null) {
+          return;
+        }
+        const interval = 3000;
+        qx.event.Timer.once(() => this.__waitForServiceReady(srvUrl), this, interval);
+      };
+
+      // ping for some time until it is really reachable
+      try {
+        const response = await fetch(srvUrl);
```

Comment:
Is there no follow redirects option for fetch?
We usually have that in the python clients

Reply:
It is properly followed later on. The only purpose in that call is to check weather the url is reachable